thoth/storages/graph/postgres.py

```diff
@@ -4198,7 +4199,7 @@ def sync_analysis_result(self, document: dict) -> None:
                 image_sha=document["result"]["layers"][-1],
                 os_name=os_name,
                 os_version=os_version,
-                cuda_version=cuda_version,
+                cuda_version=cuda_nvcc_version,
```

Comment:
What about saving the one found?

```suggestion
                cuda_version=cuda_nvcc_version or cuda_found_in_file_version,
```

Reply:
Nice, thanks @fridex !